readthedocs/proxito/views/serve.py

```diff
@@ -357,6 +358,25 @@ def get_using_unresolver(self, request):
             # TODO: find a better way to pass this to the middleware.
             request.path_project_slug = exc.project.slug
             raise Http404
+        except TranslationWithoutVersionError as exc:
```

Comment:
It seems we are using `Translation...` everywhere, but I think a more accurate word here would be `Language` instead. I think it's too late to suggest this change, but I wanted to note it since I just realized this.